packages/core-modules/.solcover.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  skipFiles: ['contracts/Proxy.sol', 'contracts/Router.sol'],
+  skipFiles: ['contracts/Router.sol'],
```

Comment:
We don't need to skip the Proxy because its empty, so it doesn't add uncovered lines to the coverage report.